src/main/drivers/flash_impl.h

```diff
@@ -28,8 +28,23 @@
 
 struct flashVTable_s;
 
+typedef enum {
+    FLASHIO_NONE = 0,
+    FLASHIO_SPI,
+    FLASHIO_QUADSPI
+} flashDeviceIoMode_e;
+
+typedef struct flashDeviceIO_s {
+    union {
+        busDevice_t *busdev; // Device interface dependent handle (spi/i2c)
+    #ifdef USE_QUADSPI
+        QUADSPI_TypeDef *quadSpi;
+    #endif
+    } handle_u;
```

Comment:
If we are to use nested type definitions, this should not be snake case.

Reply:
`handle_u`? How do you want this to be and why?

Reply:
The `_u` suffix is normally used for type definitions, not for structure members. Since this does not use a type definition, the structure member should be `handle`.

Reply:
Ah...

Reply:
Done